doc/nrf/releases_and_maturity/releases/release-notes-3.0.0.rst

```diff
@@ -0,0 +1,1441 @@
+    The workaround is already included in the MPSL component, so the solution is applicable only if your application does not use MPSL.
+
+Developing with nRF54H Series
+=============================
+
+* Added a new documentation page :ref:`ug_nrf54h20_suit_signing` under :ref:`ug_nrf54h20_suit_dfu`.
+* Removed the note on installing SEGGER USB Driver for J-Link on Windows from the :ref:`ug_nrf54h20_gs` page and moved its contents to the `nRF Util prerequisites`_ documentation.
+  The Windows-only requirement to install the SEGGER USB Driver for J-Link is now mentioned in the :ref:`installing_vsc` section on the :ref:`installation` page.
+
+Developing with Front-End Modules
+=================================
+
+* Added support for the following:
+
+  * :ref:`nRF2220 Front-End Module <ug_radio_fem_nrf2220>`.
+  * :ref:`nRF2220 EK shield <ug_radio_fem_nrf2220ek>`.
+  * :ref:`nRF21540 Front-End Module in GPIO mode <ug_radio_fem_nrf21540_gpio>` for the nRF54L Series devices.
+
+* Fixed an issue for the nRF21540 Front-End Module (for GPIO and GPIO+SPI modes) when spurious emission occurred due to late activation of the ``TX_EN`` pin.
+  The ``PDN`` pin is now activated earlier by the call to the :c:func:`mpsl_fem_enable` function.
+  The ``TX_EN`` pin is now activated 15 µs earlier during the ramp-up of the radio.
+
+Developing with PMICs
+=====================
+
+* Added the :ref:`ug_npm2100_developing` documentation.
+
+Security
+========
+
+* Added:
+
+  * Support for HKDF-Expand and HKDF-Extract in CRACEN.
+  * Support for HashEdDSA (ed25519ph) to CRACEN.
+  * TF-M now supports Attestation service on nRF54L15
+  * The following documentation pages:
+
+    * :ref:`ug_tfm_architecture`.
+    * :ref:`ug_psa_certified_api_overview`.
+    * :ref:`ug_tfm_supported_services`.
+
+* Updated:
+
+  * The Oberon PSA core to version 1.3.4 that introduces support for the following:
+
+    * PSA static key slots with the option :kconfig:option:`CONFIG_MBEDTLS_PSA_STATIC_KEY_SLOTS`.
+    * NIST SP 800-108 conformant CMAC and HMAC based key derivation using Oberon PSA driver.
+
+      For more information regarding the Oberon PSA core v1.3.4 update, see the relevant changelog entry in the `Oberon PSA core changelog`_.
+
+    * The :ref:`app_approtect` page with nRF Util commands that replaced the nrfjprog commands.
+      This is part of the ongoing work of archiving `nRF Command Line Tools`_ and replacing them with nRF Util.
+    * The :ref:`app_boards_spe_nspe` documentation page from the :ref:`ug_app_dev` section has been moved under :ref:`ug_tfm_index`.
+
+* Removed the Running applications with Trusted Firmware-M page.
+  Its contents have been moved into the following new pages:
+
+  * :ref:`ug_tfm_index`
+  * :ref:`ug_tfm_building`
+  * :ref:`ug_tfm_logging`
+  * :ref:`ug_tfm_services`
+  * :ref:`ug_tfm_provisioning`
+
```

Comment:
Please add:

west ncs-provision command was ported onto newest nrfutil device provisioning command. User must update `nrfutil; device` to version recommended for this NCS release.

Reply:
Added under Build and Configuration.